packages/react-native-drawer-layout/src/views/Drawer.tsx

```diff
@@ -28,12 +28,8 @@ import {
 } from '../constants';
 import type { DrawerProps } from '../types';
 import { DrawerProgressContext } from '../utils/DrawerProgressContext';
-import {
-  GestureHandlerRootView,
-  GestureState,
-  PanGestureHandler,
-  type PanGestureHandlerGestureEvent,
-} from './GestureHandler';
+import { GestureHandlerRootView, GestureState } from './GestureHandler';
+import { GestureDetector } from './GestureHandlerNative';
```

Comment:
Don't import from `GestureHandlerNative.tsx` directly. Import from `GestureHandler.tsx` so that the appropriate implementation is imported based on the platform.